### PR TITLE
Improve error handling about bundle command

### PR DESCRIPTION
**Pull request by kenhys**
Closes: #193

### CODE REVIEW

**Comment by kenhys** [comment]
On windows, concurrent-ruby not found. (but td-agent/downloads/concurrent-ruby-1.1.6.gem exists) ???

```
Updating files in C:/work/td-agent-builder-msi/td-agent/downloads
Could not find concurrent-ruby-1.1.7.gem for installation
PS C:\work\td-agent-builder-msi\td-agent> echo $?
False
```

**Comment by kenhys** [comment]
build error is not reproducible on local PC.

**Comment by kenhys** [comment]
centos6 console 1.9.0 not compatible ruby 24
centos7 ni gem install
centos8 nil gem install
buster nil gem install
xenial no _2.2.0.rc.1_
bionic no _2.2.0.rc.1_
focal no rake 13.0.1

**Comment by kenhys** [comment]
> centos6 console 1.9.0 not compatible ruby 24

Need to downgrade some gem files.

**Comment by kenhys** [comment]
CI has passed except Travis-CI.

File: td-agent/Rakefile [hunk 1]
```diff
@@ -868,13 +866,23 @@ class BuildTask
...
-    sh(bundle_command, "config", "set", "--local", "cache_path", relative_path)
-    sh(bundle_command, "package", "--no-install")
+    gem_home = ENV["GEM_HOME"]
+    ENV["GEM_HOME"] = gem_staging_dir
+    sh(bundle_command, "_2.2.0.rc.1_", "config", "set", "--local", "cache_path", relative_path)
+    sh(bundle_command, "_2.2.0.rc.1_", "package", "--no-install")
+    ENV["GEM_HOME"] = gem_home
 
+    Dir.glob("**/*.gem").sort.each do |gem_path|
+      puts "fetched gem path: <#{gem_path}>"
```
Comment (cosmo0920) [inline td-agent/Rakefile:876]: nits:
How about using [logger](https://docs.ruby-lang.org/ja/latest/class/Logger.html) to specify log level in the later PR?

**Review by cosmo0920 — COMMENTED** [review]
I'd confirmed to build apt:build, msi:build, and dmg:selfbuild.
I'm confirming to build yum:build.

**Review by cosmo0920 — APPROVED** [review]

File: td-agent/Rakefile (hunk 1 above)
Comment (kenhys) [reply td-agent/Rakefile:876]: It seems reasonable!

**Pull request merged**